pygmt/helpers/decorators.py

```diff
@@ -13,111 +13,94 @@
-    "XY": """\
-        xshift : str
-            ``[a|c|f|r][xshift]``.
-            Shift plot origin in x-direction.
-        yshift : str
-            ``[a|c|f|r][yshift]``.
-            Shift plot origin in y-direction. Full documentation is at
-            :gmt-docs:`gmt.html#xy-full`.
-         """,
-    "j": """\
-        distcalc : str
-            ``e|f|g``.
-            Determine how spherical distances are calculated.
-
-            - **e** - Ellipsoidal (or geodesic) mode
-            - **f** - Flat Earth mode
-            - **g** - Great circle distance [Default]
-
-            All spherical distance calculations depend on the current ellipsoid
-            (PROJ_ELLIPSOID), the definition of the mean radius
-            (PROJ_MEAN_RADIUS), and the specification of latitude type
-            (PROJ_AUX_LATITUDE). Geodesic distance calculations is also
-            controlled by method (PROJ_GEODESIC).""",
-    "n": """\
-        interpolation : str
-            ``[b|c|l|n][+a][+bBC][+c][+tthreshold]``
-            Select interpolation mode for grids. You can select the type of
-            spline used:
-
-            - 'b' for B-spline
-            - 'c' for bicubic [Default]
-            - 'l' for bilinear
-            - 'n' for nearest-neighbor""",
-    "p": """\
-        perspective : list or str
-            ``'[x|y|z]azim[/elev[/zlevel]][+wlon0/lat0[/z0]][+vx0/y0]'``.
-            Select perspective view and set the azimuth and elevation angle of
-            the viewpoint. Default is [180, 90]. Full documentation is at
-            :gmt-docs:`gmt.html#perspective-full`.
-        """,
-    "registration": """\
-        registration : str
-            ``[g|p]``
-            Force output grid to be gridline (g) or pixel (p) node registered.
-            Default is gridline (g).""",
-    "t": """\
-        transparency : float
-            Set transparency level, in [0-100] percent range.
-            Default is 0, i.e., opaque.
-            Only visible when PDF or raster format output is selected.
-            Only the PNG format selection adds a transparency layer
-            in the image (for further processing). """,
-    "x": """\
-        cores : int
-            ``[[-]n]``.
-            Limit the number of cores to be used in any OpenMP-enabled
-            multi-threaded algorithms. By default we try to use all available
-            cores. Set a number *n* to only use n cores (if too large it will
-            be truncated to the maximum cores available). Finally, give a
-            negative number *-n* to select (all - n) cores (or at least 1 if
-            n equals or exceeds all).
-            """,
+    "R": r"""region : str or list
+    *xmin/xmax/ymin/ymax*\ [**+r**\ ][**+u**\ *unit*].
+    Specify the region of interest. This is a required argument if this
+    is the first plot command.""",
+    "J": r"""projection : str
+    *projection*\ [*projection-specific arguments*\ ]\ *figure size*.
```

Comment:
**-J** option is very difficult to read.
![image](https://user-images.githubusercontent.com/3974108/107436714-ec604600-6afb-11eb-95d4-966a8c36b819.png)